### PR TITLE
Introduce cases for fixedsalinity

### PR DESCRIPTION
**Pull request by mcwimm**
@jvollhueter @zaphod-bx 
based on our discussion this week, I added "cases" to the fixed salinity modules. Would this fit your requirements?

### CODE REVIEW

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.md
Comment (zaphod-bx) [inline ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.md:1]: it is not an "exponential" function. I would suggest "sigmoidal salinity stress factor" ?

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py [hunk 1]
```diff
@@ -173,7 +190,7 @@ def getInputParameters(self, args):
                 print("WARNING: Tag " + tag + " not specified for " + case +
                       " below-ground " + "initialisation!")
 
-        if len(missing_tags) > 0:
+        if len(missing_tags) > 1:
```
Comment (zaphod-bx) [inline ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py:193]: works! But isn't there the danger, that variant is given, but anothere missing?

**Review by zaphod-bx — CHANGES_REQUESTED** [review]

**Comment by zaphod-bx** [comment]
I think it doesn't fit my requirements. At least I don't see how. I would need that the tree gets a variable "salinity" with a value of average salinity in the root zone for estimating the required vessel diameter distribution (in the tree - module).

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py [hunk 2]
```diff
@@ -41,10 +48,16 @@ def calculateBelowgroundResources(self):
...
         salinity_plant = self.getPlantSalinity()
-        psi_zero = np.array(self._psi_leaf) + (2 * np.array(self._r_crown) +
-                                               np.array(self._h_stem)) * 9810
-        psi_sali = np.array(psi_zero) + 85000000 * salinity_plant
-        self.belowground_resources = psi_sali / psi_zero
+
+        if self.variant is None or self.variant == "bettina":
+            psi_zero = np.array(self._psi_leaf) + (2 * np.array(self._r_crown) +
+                                                   np.array(self._h_stem)) * 9810
+            psi_sali = np.array(psi_zero) + 85000000 * salinity_plant
```
Comment (zaphod-bx) [inline ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py:55]: these 3 lines is what I have to reverse in the tree module to conclude back on "salinity_plant"

Comment (mcwimm) [reply ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py:55]: So you need the salinity value to be transfered between modules? If so, the value can just be saved as other growth variables.

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.md
Comment (mcwimm) [reply ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.md:1]: fixed

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py (hunk 2 above)
Comment (mcwimm) [reply ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py:55]: Maybe it`s not a good idea to transfer salinity directly....

**Review by jvollhueter — APPROVED** [review]
ready to merge

**Review by jvollhueter — APPROVED** [review]
ready

File: ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py (hunk 2 above)
Comment (jvollhueter) [reply ResourceLib/BelowGround/Individual/FixedSalinity/FixedSalinity.py:55]: .

**Review by zaphod-bx — APPROVED** [review]

**Pull request merged**